### PR TITLE
feat: simplify the cucumber parser

### LINKED ISSUE #71: FR: Create a new CucumberJS parser that replaces both CucumberJS parsers
**Current situation:**
Currently protractor-flake for CucumberJS can use 2 parsers, the [cucumber.js](https://github.com/NickTomlin/protractor-flake/blob/master/src/parsers/cucumber.js) and the [cucumber.multi.js](https://github.com/NickTomlin/protractor-flake/blob/master/src/parsers/cucumber.multi.js) parser. The first parser is based on the logging of the framework for CucumberJS < 0.9.0 and the second one is based on a `console.log`.

**Desired situation:**
Replace both parsers with a new parser that will read the log based on a `console.log` users need to add to their framework. This is currently already [advised](https://github.com/NickTomlin/protractor-flake/blob/master/docs/cucumber.md#always-print-specs) for CucumberJS 1.
The advantage of parsing the log of protractor based on a `console.log` is that it will be more stable and framework independent then the current parsing because that one is based on the logging of the framework itself.

**To do:**
 - [] create a new parser called `cucumber`, the other ones will be removed and this will result in a **new major release**
 - [] test this for CucumberJS 1, 2 and 3
 - [] create new unit tests
 - [] create new docs for cucumber with examples how to implement this for CucumberJS 1, 2 and 3

### PR DESCRIPTION
**Pull request by wswebcreation**
This change simplifies the cucumber parsers and brings them down to 1 parser.
The docs are also updated with usage examples.

Closes #71

BREAKING CHANGE: cucumberMulti parser can't be used anymore

To migrate change `cucumberMulti` to `cucumber`

### CODE REVIEW

File: src/parsers/cucumber.js [hunk 1]
```diff
@@ -1,14 +1,21 @@
 export default {
-  name: 'cucumber',
-
+  name: 'cucumberMulti',
```
Comment (NickTomlin) [inline src/parsers/cucumber.js]: Should this name just be cucumber?

**Review by NickTomlin — APPROVED** [review]
💃 
Looks great! I made one small comment about the parser name and I can merge this in right after you take care of that

**Comment by wswebcreation** [comment]
Thanks for seeing this. I've changed the name. Strangely enough it came through all the checks, even mine :(.

**Review by NickTomlin — APPROVED** [review]
💃

**Comment by wswebcreation** [comment]
@NickTomlin 

Are you going to create a new major version? Because git and NPM are not equal anymore.

**Comment by NickTomlin** [comment]
@wswebcreation  🤦‍♂️ ugh it looks like this didn't get automatically published. I've gone ahead and released 3.0.0 manually

**Comment by wswebcreation** [comment]
@nicktomlin

Shit happens 😂, tnx for releasing it

**Pull request merged**